context/book/primary-adapter/rest/spring-web-mvc/build.gradle.kts

```diff
@@ -6,6 +6,8 @@ dependencies {
     implementation(project(":context:book:domain"))
 
     implementation("org.springframework.boot:spring-boot-starter-web:2.3.3.RELEASE")
+    implementation(project(":context:book:domain"))
```

Comment:
IMHO it should be nicer to group "implementation" and "implementation(project..." together. It is a matter to group by implementation type.

Reply:
totally true, it's easier to see it